src/ECModel.jl

```diff
@@ -852,6 +853,187 @@ function split_financial_terms(ECModel::AbstractEC, profit_distribution=nothing)
+
+    # Investment costs
+    CAPEX = JuMP.Containers.DenseAxisArray(
+        [(y==1) ? sum(Float64[get_value(ECModel.results[:CAPEX_tot_us], u)]) : 0.0
+            for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+           , year_set, user_set
+    )
+    # Maintenance costs
+    Ann_Maintenance = JuMP.Containers.DenseAxisArray(
+        [get_value(ECModel.results[:C_OEM_tot_us], u)
+            for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+        ]
+        , year_set, user_set
+    )
+    # Replacement costs
+    Ann_Replacement = JuMP.Containers.DenseAxisArray(
+            [get_value(ECModel.results[:C_REP_tot_us][y, :], u)
+                for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+            , year_set, user_set
+     )
+    # Recovery value
+    Ann_Recovery = JuMP.Containers.DenseAxisArray(
+        [get_value(ECModel.results[:R_RV_tot_us][y, :], u)
+            for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+                , year_set, user_set
+    )
+
+    # Peak energy charges
+    Ann_peak_charges = JuMP.Containers.DenseAxisArray(
+        [get_value(ECModel.results[:C_Peak_tot_us], u)
+            for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+                , year_set, user_set
+    )
+
+    # Get revenes by selling energy and costs by buying or consuming energy
+    zero_if_negative = x->((x>=0) ? x : 0.0)
+    Ann_energy_revenues = JuMP.Containers.DenseAxisArray(
+        [(u in axes(ECModel.results[:R_Energy_us])[1]) ? sum(zero_if_negative.(ECModel.results[:R_Energy_us][u,:])) : 0.0
+            for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+            , year_set, user_set
+    )
+    Ann_energy_costs = JuMP.Containers.DenseAxisArray(
+        [(u in axes(ECModel.results[:R_Energy_us])[1]) ? sum(zero_if_negative.(.-(ECModel.results[:R_Energy_us][u, :]))) : 0.0
+        for y in year_set, u in setdiff(user_set_financial, [EC_CODE])
+            ]
+            , year_set, user_set
+    )
+    Ann_net_energy_costs = Ann_energy_costs .- Ann_energy_revenues
+    
+    # Total OPEX costs
+    OPEX = Ann_Maintenance .+ Ann_peak_charges .+ Ann_net_energy_costs
+
+    # get NPV given the reward allocation
+    #Check how to proceed
+    NPV = profit_distribution
+
+    # Total reward
+    Ann_reward = NPV .+ CAPEX .+ OPEX .+ Ann_Replacement .- Ann_Recovery
+    
+    return (
+        NPV=NPV,
+        CAPEX=CAPEX,
+        OPEX=OPEX,
+        OEM = Ann_Maintenance,
+        REP=Ann_Replacement,
+        RV=Ann_Recovery,
+        REWARD=Ann_reward,
+        PEAK=Ann_peak_charges,
+        EN_SELL=Ann_energy_revenues,
+        EN_CONS=Ann_energy_costs,
+        EN_NET=Ann_net_energy_costs,
+    )
+end
+
+function business_plan_dataframe(ECModel::AbstractEC,profit_distribution=nothing, user_set_financial=nothing)

```

Comment:
Please add the docstring.

Shall the function be called simply "business_plan"? what do you think?